src/ovp_pipeline/context_binder.py

```
@@ -0,0 +1,679 @@
+    render inside ``<!-- context-manifest ... -->``).
+    """
+    lines: list[str] = []
+    lines.append(f"context_built_at: {m.context_built_at}")
+    lines.append(f"token_estimate: {m.token_estimate_total}")
+    lines.append(f"anchor_kind: {m.anchor.kind}")
+    if m.anchor.ref:
+        lines.append(f"anchor_ref: {m.anchor.ref}")
+    if m.anchor.included_evergreens:
+        lines.append("included_evergreens:")
+        for slug in m.anchor.included_evergreens:
+            lines.append(f"  - {slug}")
```

Ruff (0.15.12)

[warning] Use `list.extend` to create a transformed list

(PERF401)